plugins/woocommerce/tests/performance/requests/shopper/cart-apply-coupon.js

```diff
@@ -109,43 +112,44 @@ export function cartApplyCoupon() {
 		);
 
 		response = http.post(
-			`${base_url}/?wc-ajax=apply_coupon`,
+			`${ base_url }/?wc-ajax=apply_coupon`,
 			{
-				coupon_code: `${coupon_code}`,
-				security: `${apply_coupon_nonce}`,
+				coupon_code: `${ coupon_code }`,
+				security: `${ apply_coupon_nonce }`,
 			},
 			{
 				headers: requestheaders,
+				tags: { name: 'Shopper - wc-ajax=apply_coupon' },
 			}
 		);
 
-		applyCouponTrend.add(response.timings.duration);
-		check(response, {
-			"is status 200": (r) => r.status === 200,
-			"body contains: 'Coupon code applied successfully'": (response) =>
-				response.body.includes("Coupon code applied successfully"),
-		});
+		check( response, {
+			'is status 200': ( r ) => r.status === 200,
+			"body contains: 'Coupon code applied successfully'": ( response ) =>
+				response.body.includes( 'Coupon code applied successfully' ),
```

Comment:
This assertion is failing when I run things locally.

Reply:
Good catch, thanks for flagging this! This request is intended for use in scenarios for testing sites that already have coupons setup, which can then be added to the `coupon_code` in `config.js`. It isn't included in the CI test scenarios for this reason.
We have an existing issue for Coupon creation here https://github.com/woocommerce/woocommerce/issues/32315

I think this is ok as the thing would happen if running with `trunk` right now.